index.html

```diff
@@ -1,59 +1,99 @@
-            </ul>
-        </div>
-        <div class="middle">
-            <h1>Hey There. I'm Alex</h1>
-            <h3>I' am a software Developer</h3>
-            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</p>
-
-        </div>
-        <div class="right-side">
-            <svg class="img" width="419" height="358" viewBox="0 0 419 358" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path d="M266.898 331.677L299.986 322.505L290.819 289.433L257.731 298.605L248.564 265.533L281.652 256.362L272.485 223.29L239.397 232.461L206.309 241.633L197.142 208.561L230.23 199.389L221.063 166.317L187.975 175.489L197.142 208.561L164.054 217.732L130.966 226.904L121.799 193.832L88.7108 203.003L55.6227 212.175L64.7897 245.247L73.9567 278.318L40.8686 287.49L31.7016 254.418L16.9393 258.51L1.49656 35.6899C1.49656 35.6899 23.4178 3.04098 82.3162 0.960448C170.537 -2.12506 213.836 45.3191 286.845 61.5168C332.729 71.81 342.908 51.1822 342.908 51.1822L372.679 242.636C372.679 242.636 449.471 226.008 493.319 277.409C519.34 308.001 480.228 339.936 439.759 351.153C397.508 362.864 347.536 355.622 267.533 333.966L266.898 331.677Z" fill="white" stroke="#FF6B00" stroke-width="1.22338" stroke-miterlimit="10"/>
-            </svg>
-                
-        </div>
-    </main>
-    
-</body>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Personal Portofolio</title>
+
+    <!-- favicon -->
+    <link rel="stylesheet" href="images/favicon-1.ico" type="image/x-icon" />
+    <!-- font awesome -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+        integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+    <!-- Header -->
+    <header class="header">
+        <a href="" class="logo">john Doe</a>
+        <div class="nav-links">
+           <ul>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+            <li><a href="#">Home</a></li>
+           </ul>
+        </div>
+        <span class="nav-menu-icon"><i class="fa-solid fa-bars"></i></span>
+    </header>
+    <div class="image">
+            <img src="./images/Ellipse18.png" alt="#">
+    </div>
+    <div class="image-2">
+           <img src="./images/Ellipse19.png" alt="#">
+    </div>

```

Comment:
Since decorations are always background images, not `img` tags kindly use these images as `background-images` in your CSS file.